src/org/parosproxy/paros/view/View.java

```diff
@@ -168,6 +175,8 @@
     // ZAP: splash screen
     private SplashScreen splashScreen = null;
 
+    private static Map<AddOn.Status, StatusUI> statusMap = new HashMap<AddOn.Status, StatusUI>();

```

Comment:
Not remove everything, it should be kept the diamond:
`Map<AddOn.Status, StatusUI> statusMap = new HashMap<>();`

the compiler infers the types for us (in this case.)